### PR TITLE
Translate React Team Principles into Traditional Chinese

### PR DESCRIPTION
**Pull request by cwenwen**
The pull request contains a translation of the article “What Are the React Team Principles?” to Traditional Chinese.

### CODE REVIEW

File: src/pages/what-are-the-react-team-principles/index.zh-hant.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+我們不想卡在 local maxima，如果有某種做法在理論上更站得住腳，就算會花上好幾年，我們也樂於投注心力在其之上。在達成目標的過程中會遇到許多障礙，有時必須因現實考量而妥協，但我們相信若持續地克服這些困難，理論終究會獲勝。
+
+## 你們團隊的準則是什麼？
+
+以上是我觀察到的 React 團隊在工作時的基本原則，但我可能漏了很多。我也還沒提到 React 如何推出 API、團隊如何溝通未來的改動方向等等。或許下次可以來談談這些。
+
+你們團隊也有一系列的工作準則嗎？我洗耳恭聽。
+
+_原始文章發表在[這裡](https://react.christmas/2019/24)._
```
Comment (neighborhood999) [inline src/pages/what-are-the-react-team-principles/index.zh-hant.md]:
```suggestion
_原始文章發表在[這裡](https://react.christmas/2019/24)。_
```

File: src/pages/what-are-the-react-team-principles/index.zh-hant.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+spoiler: UI 優先於 API
+---
+
+我很幸運地在 React 團隊的這段時間內，看見 [Jordan](https://twitter.com/jordwalke)、[Sebastian](https://twitter.com/sebmarkbage)、[Sophie](https://twitter.com/sophiebits) 和其他團隊成員如何解決問題。在這篇文章中，我會把從他們身上所學到的，濃縮為一些較高層級的技術準則。這些準則未必詳盡，它們只是我個人對 React 團隊運作的觀察和整理——其他團隊成員或許會有不同的觀點。
+
+## UI 優先於 API
+
+每個抽象概念合理地部署時，都有其古怪之處。這些地方該如何在 UI 上清楚呈現？你能看出一個應用程式中蘊含的特定抽象概念嗎？
+
+抽象概念對使用者體驗有直接的影響——它能創造好的、延續性的體驗或限制某些操作。這使得我們設計 API 時，並不會從抽象概念開始發想。我們會從想呈現的使用者體驗來著手，之後再走回抽象概念。
```
Comment (kpman) [inline src/pages/what-are-the-react-team-principles/index.zh-hant.md]: origin: `之後再走回抽象概念。`
I prefer: `之後再回歸抽象概念。`

Comment (kpman) [reply src/pages/what-are-the-react-team-principles/index.zh-hant.md]:
```suggestion
抽象概念對使用者體驗有直接的影響——它能創造好的、延續性的體驗或限制某些操作。這使得我們設計 API 時，並不會從抽象概念開始發想。我們會從想呈現的使用者體驗來著手，之後再回歸抽象概念。
```

File: src/pages/what-are-the-react-team-principles/index.zh-hant.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+有時當我們回到了抽象概念，會發現必須換掉整個作法，才能打造正確的使用者體驗。如果我們從 API 下手，我們就無法察覺到這點。所以我們把 UI 的優先性放在 API 前面。
```
Comment (neighborhood999) [inline src/pages/what-are-the-react-team-principles/index.zh-hant.md]:
```suggestion
有時當我們回到了抽象概念，會發現必須換掉整個作法，才能打造正確的使用者體驗。如果我們從 API 下手，就無法察覺到這點。UI 的權重應該大於 API。
```

File: src/pages/what-are-the-react-team-principles/index.zh-hant.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+## 吸收複雜性
+
+React 內部程式碼的簡單化並不是我們的目標。為了使產品開發者使用 React 寫出更平易近人、易於改動的程式碼，我們樂於複雜化 React 內部的實作。
```
Comment (neighborhood999) [inline src/pages/what-are-the-react-team-principles/index.zh-hant.md]:
```suggestion
簡化 React 內部程式碼的並不是我們的目標。為了讓產品開發者可以使用 React 寫出更平易近人、易於改動的程式碼，我們樂於複雜化 React 內部的實作。
```

File: src/pages/what-are-the-react-team-principles/index.zh-hant.md (hunk 3 above)
Comment (cwenwen) [reply src/pages/what-are-the-react-team-principles/index.zh-hant.md]: How about `所以我們先考慮 UI，再考慮 API` ?

Comment (neighborhood999) [reply src/pages/what-are-the-react-team-principles/index.zh-hant.md]: 👌

**Pull request merged**